pkgs/top-level/python-packages.nix

```diff
@@ -8195,22 +8201,8 @@ in {
 
   pyte = callPackage ../development/python-modules/pyte { };
 
-  graphviz = buildPythonPackage rec {
-    name = "graphviz-${version}";
-    version = "0.5.2";
-
-    src = pkgs.fetchurl {
-      url = "mirror://pypi/g/graphviz/${name}.zip";
-      sha256 = "0jh31nlm0qbxwylhdkwnb69pcjlc5z03fcfbs0gvgzp3hfrngsk0";
-    };
-
-    propagatedBuildInputs = [ pkgs.graphviz ];
-
-    meta = {
-      description = "Simple Python interface for Graphviz";
-      homepage = https://github.com/xflr6/graphviz;
-      license = licenses.mit;
-    };
+  graphviz = callPackage ../development/python-modules/graphviz {
+    pkgs_graphviz = pkgs.graphviz;
```

Comment:
no need to call it `pkgs_graphviz`, just call the parameter `graphviz` and use `inherit (pkgs) graphviz`

Reply:
Done.